internal/controllers/server/tests/server-import/02-assert.yaml

```diff
@@ -20,10 +20,9 @@ metadata:
 status:
   conditions:
     - type: Available
-      # Workaround for https://github.com/k-orc/openstack-resource-controller/issues/280
-      # The status should be True
-      message: OpenStack resource is up to date
-      status: "False"
+      # Commented out because of https://github.com/k-orc/openstack-resource-controller/issues/280
+      # message: OpenStack resource is available
+      # status: "True"
       reason: Success
```

Comment:
I'm pretty sure that asserting `reason: Success` here is equivalent to asserting `status: True`, so I think you don't want that.